packages/html-reporter/src/reportView.tsx

```diff
@@ -50,6 +54,8 @@ export const ReportView: React.FC<{
   const [filterText, setFilterText] = React.useState(searchParams.get('q') || '');
   const [metadataVisible, setMetadataVisible] = React.useState(false);
 
+  const gitCommitInfo = React.useMemo(() => filterMetadata(report?.json().metadata || {}).gitCommitInfo, [report]);
```

Comment:
I'd hold it more generic and do it for Metadata instead. this means we don't need to touch filterMetadata and can call the context MetadataContext I think.